test/minitest5.bats

```diff
@@ -31,9 +31,9 @@ setup() {
 
 @test "multi-master succeeds when all tests pass" {
   export TEST_QUEUE_RELAY_TOKEN=$(date | cksum | cut -d' ' -f1)
-  TEST_QUEUE_SOCKET=0.0.0.0:12345 bundle exec minitest-queue ./test/samples/sample_minitest5.rb &
+  TEST_QUEUE_RELAY=0.0.0.0:12345 bundle exec minitest-queue ./test/samples/sample_minitest5.rb || true &
```

Comment:
Not worth coming back to on its own, but this might worth docs the next time you touch this file. This `|| true` would surprise me.